Makefile

```diff
@@ -165,7 +166,8 @@ test/addons/.buildstamp: config.gypi \
 	# Cannot use $(wildcard test/addons/*/) here, it's evaluated before
 	# embedded addons have been generated from the documentation.
 	for dirname in test/addons/*/; do \
```

Comment:
@thefourtheye No strong feelings from my side. Was more if the goal was to reduce the amount of output.